cpp/src/copying/slice.cpp

```diff
@@ -33,7 +33,7 @@ std::vector<column_view> slice(column_view const& input,
 
   std::vector<column_view> result{};
 
-  if (indices.size() == 0 or input.size() == 0) { return result; }
+  if (indices.size() == 0) { return result; }
```

Comment:
Recommend using `empty` and moving `result` towards the bottom and just return `{}` here.
```suggestion
  if (indices.empty()) return {};
```